src/components/pages/dashboard/panel/panelErrorBoundary.js

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import React, { Component } from 'react';
+
+import { Panel } from './panel';
+import { PanelError } from './panelError';
+
+export class PanelErrorBoundary extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { hasError: false };
+  }
+
+  componentDidCatch(error, info) {
+    this.setState({ hasError: true });
+  }
+
+  render() {
+    return this.state.hasError
+      ? <Panel>
+          <PanelError>{ this.props.msg }</PanelError>
```

Comment:
Should this use ErrorMsg?

Reply:
Potentially but I want to confirm dashboard panel error messages with design.